app/src/main/java/com/example/bigowlapp/repository/exception/PhoneNumberTakenException.java

```diff
@@ -0,0 +1,14 @@
+package com.example.bigowlapp.repository.exception;
+
+public class PhoneNumberTakenException extends Exception {
+
+    private static final String DEFAULT_MESSAGE = "Phone Number is already in the database.";
```

Comment:
You should change the message to not mention "database", since this message is seen by a user, and the database has no relevance to them.

The message should simply tell the user that the phone number is already in use taken, or that an account is already registered with the number. Anything along those lines

Reply:
Yeah, I'll change that

Reply:
Yeah, that's reasonable.